src/users/users.service.ts

```diff
@@ -52,36 +56,73 @@ export async function createUser({
   name,
   metadata
 }: Pick<UserInput, 'externalId' | 'email' | 'name'> & UserCreateBody): Promise<UserCreateResponse> {
-  const user = new User({ externalId, email, name, metadata });
+  const user = new User({
+    externalId,
+    email,
+    name,
+    metadata,
+    defaultOrganization: ORM.em
+      .getRepository(Organization)
+      .getReference('placeholder', { wrapped: true }),
+    defaultProject: ORM.em.getRepository(Project).getReference('placeholder', { wrapped: true })
+  });
 
-  const defaultOrgOwner = await ORM.em
-    .getRepository(OrganizationUser)
-    .findOneOrFail(
-      { id: ORGANIZATION_OWNER_ID_DEFAULT },
-      { filters: { orgAdministrationAccess: false } }
-    );
-  const defaultProjectAdmin = await ORM.em
-    .getRepository(ProjectPrincipal)
-    .findOneOrFail(
-      { id: PROJECT_ADMIN_ID_DEFAULT },
-      { filters: { projectAdministrationAccess: false } }
-    );
+  let orgUser: OrganizationUser | null = null;
+  let organization: Organization | null = null;
+  if (email && email.endsWith('@ibm.com')) {
```

Comment:
This is not enough. Valid IBM emails are also:
`partner.ibm.com`, `us.ibm.com`, `cz.ibm.com`, etc.

Reply:
OK, can we go only with `endsWith('ibm.com')`?

Reply:
I don't think so, consider `foobar@foobarbaztestibm.com`

Reply:
what about `endsWith('.ibm.com')` ?